homeassistant/components/vicare/sensor.py

```diff
@@ -210,6 +210,62 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         unit_getter=lambda api: api.getGasConsumptionHeatingUnit(),
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    ViCareSensorEntityDescription(
+        key="gas_consumption_fuelcell_today",
+        translation_key="gas_consumption_fuelcell_today",
+        value_getter=lambda api: api.getFuelCellGasConsumptionToday(),
+        unit_getter=lambda api: api.getFuelCellGasConsumptionUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="gas_consumption_fuelcell_this_week",
+        translation_key="gas_consumption_fuelcell_this_week",
+        value_getter=lambda api: api.getFuelCellGasConsumptionThisWeek(),
+        unit_getter=lambda api: api.getFuelCellGasConsumptionUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="gas_consumption_fuelcell_this_month",
+        translation_key="gas_consumption_fuelcell_this_month",
+        value_getter=lambda api: api.getFuelCellGasConsumptionThisMonth(),
+        unit_getter=lambda api: api.getFuelCellGasConsumptionUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="gas_consumption_fuelcell_this_year",
+        translation_key="gas_consumption_fuelcell_this_year",
+        value_getter=lambda api: api.getFuelCellGasConsumptionThisYear(),
+        unit_getter=lambda api: api.getFuelCellGasConsumptionUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="gas_consumption_total_today",
+        translation_key="gas_consumption_total_today",
+        value_getter=lambda api: api.getGasConsumptionTotalToday(),
+        unit_getter=lambda api: api.getGasConsumptionUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="gas_consumption_total_this_week",
+        translation_key="gas_consumption_total_this_week",
```

Comment:
From a user perspective, I would like to have access to the total gas consumption by default. A more specific consumption like "fuel cell" or "peak load boiler" (which is missing here) could be optional.

Reply:
But you don't need the consumption this week, month and year all at once

Reply:
"Today" could be enough. HA Energy dashboard can do calculations for the rest.

Reply:
> Can we maybe disable some by default?

I think so too. I suggest we disable all but the *today* sensor by default. 

I already prepared a PR for the existing entities to do so.
https://github.com/home-assistant/core/pull/109014

Reply:
@joostlek what is the recommended state, hide or disable? If the sensor is just hidden, one would already collect long term statistics for later use, right?

Reply:
I mean, what's the point of collecting LTS four times (as in, the increase is the same)

Also, the stateclass is Total increasing, while it seems like it will reset at least every day/month/year/week. I think the state class should be Total

Reply:
From the [description](https://developers.home-assistant.io/blog/2021/09/20/state_class_total/#state-class-total_increasing) `total_increasing` perfectly fits in my eyes.

Reply:
Oh you're right

Reply:
added `entity_registry_enabled_default=False`